specification/search/data-plane/Azure.Search/preview/2021-04-30-Preview/searchindex.json

```diff
@@ -1180,6 +1229,125 @@
       "additionalProperties": true,
       "description": "Captions are the most representative passages from the document relatively to the search query. They are often used as document summary. Captions are only returned for queries of type 'semantic'.."
     },
+    "AnswersDebugInfo": {
+      "properties": {
+        "queryClassifiedAsQuestion": {
+          "type": "boolean",
+          "description": "Whether the given query was classified as a question.",
+          "readOnly": true
+        }
+      },
+      "x-ms-client-name": "Answers",
+      "description": "Answers debug info for this search result."
+    },
+    "SemanticDebugInfo": {
+      "x-ms-client-name": "Semantic",
+      "description": "Semantic debug info for this search result.",
+      "$ref": "#/definitions/AnswersDebugInfo"
+    },
+    "RerankerInputResult": {
+      "properties": {
+        "title": {
+          "type": "string",
+          "readOnly": "true",
+          "description": "The title field."
+        },
+        "content": {
+          "type": "string",
+          "readOnly": "true",
+          "description": "The concatenated content fields."
+        },
+        "keywords": {
+          "type": "string",
+          "readOnly": "true",
+          "description": "The concatenated keyword fields."
+        }
+      },
+      "description": "The concatenated semantic fields sent to the reranker model.",
+      "x-ms-client-name": "RerankerInput"
+    },
+    "SemanticFieldResult": {
```

Comment:
I don't think we can reuse SemanticField since IIRC it was already defined elsewhere, I'll change the top-level "RerankerInputResult" and "SemanticFieldResult" to end with "Usage" instead of "Result".